app/src/main/java/org/mozilla/rocket/privately/ShortcutUtils.kt

```diff
@@ -11,23 +11,21 @@ import android.support.v4.content.pm.ShortcutInfoCompat
 import android.support.v4.content.pm.ShortcutManagerCompat
 import android.support.v4.graphics.drawable.IconCompat
 import org.mozilla.focus.R
-import org.mozilla.focus.utils.AppConstants
 import org.mozilla.rocket.component.LaunchIntentDispatcher
 
 class ShortcutUtils {
     companion object {
         private const val SHORTCUT_ID = "pb_shortcut"
 
         fun createShortcut(context: Context) {
```

Comment:
could you rename this to `createPrivateShortcut ` >

Reply:
> 1. Fix the background color issue when launching private mode
> 
> * adding a darker or grey background to avoid the contract ( white(launcher)->(dark)private)
> * fix the problem of launch time and make launcher activity transparent
> * if we want to start private mode directly, use an alias to avoid breaking the shortcut if we change the architecture in the future.

After the discussion with @Jacklin-ux, it might be the unsmooth animation that makes the transition weird, not the background color. For now I'll directly launch the alias activity of PrivateModeActivity. After improving the transition animation, I'll change the alias target back to the RocketLauncherActivity.